minio/kontena.yml

```diff
@@ -0,0 +1,72 @@
+stack: kontena/minio
+version: 0.2.0
+description: Minio is a distributed object storage server built for cloud applications and devops
+variables:
+  affinity:
+    type: string
+    default: label!=no-minio
```

Comment:
double negation makes me dizzy always :D does it have to use a default, could we use some `if`fing to put in the affinity if any is given?

affinity doesn't seem to be used at all